.github/workflows/test-convene-web.yml

```diff
@@ -97,22 +196,19 @@ jobs:
 
       - name: Setup test environment
         run: |
-            sudo chown -R $USER /usr/local/lib/node_modules
-            sudo chown -R $USER /usr/local/bin/
-            bin/setup
-
+          sudo chown -R $USER /usr/local/lib/node_modules
+          sudo chown -R $USER /usr/local/bin/
+          bin/setup
       - name: Start Rails server
         run: |
-            export PATH=$PATH:~/bin
-            bin/run &
-
+          export PATH=$PATH:~/bin
+          bin/run &
       - name: Run Tests
         run: |
-            # To wait for asset built
-            # TODO: Start server in production mode
-            curl --connect-timeout 5 --retry 5 --retry-delay 5 --retry-max-time 40 --retry-connrefused localhost:3000 1> /dev/null
-            bin/test
-
+          # To wait for asset built
+          # TODO: Start server in production mode
+          curl --connect-timeout 5 --retry 5 --retry-delay 5 --retry-max-time 40 --retry-connrefused localhost:3000 1> /dev/null
+          bin/test
```

Comment:
Currently, all tests are running via `bin/test` script.

In order to decompose the test job into separate jobs `test-features` and `test-rspec` that will run in parallel, the pipe is gonna need a different command to execute tests.

I think these are the appropriate commands:

- Run `test-rspec` - `bundle exec rspec`
- Run `test-features` - `yarn run test` (which invokes [package.json script test](https://github.com/zinc-collective/convene/blob/38979a307aa48810056a18db4c76b0830a6f2fff/package.json#L14))

Reply:
You could also split the `bin/test` script into `bin/test-features` and `bin/test-rspec` and then call those directly!

Reply:
maybe. But we use `bin/test` script for local test runs to conveniently run **all** the tests. If I split `bin/test` out into 2 new scripts, and we want to preserve the ability to easily run **all tests** that I'd have to... 🤔 make `bin/test` invoke `bin/test-features` and `bin/test-rspec`. Could it be an abstraction overkill?

Reply:
I think it's one of those tradeoffs where neither are advantageous over the other. Using `bin/rspec` or `bin/rake` for ruby and `yarn run test` for cucumber are totally fine.